gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -42,12 +41,52 @@ export const wrapCommand = (command) => {
     execute = async () => {
       const multisig = getContract(CONTRACT_LIST.MULTISIG, '')
       this.program = this.loadProgram(multisig.idl, multisig.programId.toString())
+
+      // Falling back on default wallet is signer is not provided
+      const signer = new PublicKey(this.flags.signer || this.wallet.payer.publicKey)
+      const rawTxs = await this.makeRawTransaction(signer)
+      // If proposal is not provided, we are at creation time, and a new proposal acc should have been created
+      const proposal = new PublicKey(this.flags.proposal || rawTxs[0].accounts[1].pubkey)
+      const latestSlot = await this.provider.connection.getSlot()
+      const recentBlock = await this.provider.connection.getBlock(latestSlot)
+      const tx = makeTx(rawTxs, {
+        recentBlockhash: recentBlock.blockhash,
+        feePayer: signer,
+      })
+      if (this.flags.execute) {
+        logger.loading(`Executing action...`)
+        const txhash = await this.provider.send(tx, [this.wallet.payer])
```

Comment:
This will fail if the `this.flags.signer` is provided with `--execute`, right?

Reply:
Yeah true. If execute is provided `signer` should fall into `wallet.payer` by default

Reply:
I'm saying we shouldn't let it fail so late, but validate input earlier and if both are set then fail with the explanation why this configuration is not possible.

Reply:
That make more sense. Changing it